packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -591,18 +591,25 @@ class TreemapNode extends TreeNode<TreemapNode> {
     this.byteSize = 0,
     this.childrenMap = const <String, TreemapNode>{},
     this.showDiff = false,
+    this.isDeferred = false,
   });
 
   final String name;
   final Map<String, TreemapNode> childrenMap;
   int byteSize;
 
   final bool showDiff;
+  final bool isDeferred;
 
   int get unsignedByteSize => byteSize.abs();
 
   Color get displayColor {
-    if (!showDiff) return mainUiColor;
+    if (!showDiff) {
+      if (isDeferred)
```

Comment:
let's change this to a ternary: `return isDeferred ? treemapDeferredColor : mainUiColor`